### ISSUE TITLE
Possible to add column to "Most Recent Maps" listing?

### ISSUE THREAD

**Issue by scottmudge** [briansd9/exile-diary#89]
Is it possible to add a column showing the total acquired XP (positive or negative, accounting for deaths) of a map/zone, in addition to the average XP per hour?

**Comment by briansd9** [comment]
Yes, I'm planning to make the displayed columns configurable. Anything else you'd like to see?

**Comment by scottmudge** [comment]
Maybe the option to display an "in-progress" version of the overlay every time returning H/O. And then the regular "final" version when switching/entering to a new instance/zone/map. 

Also perhaps the option to display an "estimated time to level" in the overlay text line, based on the avg experience gained per zone or per unit time, of the last 1-3 zones/maps/instances? Not sure if you can gleam the total experience needed to level from the client.txt log though. Just some ideas...
